packages/material-ui/src/Autocomplete/Autocomplete.js

```diff
@@ -8,7 +8,7 @@ import ListSubheader from '../ListSubheader';
 import Paper from '../Paper';
 import IconButton from '../IconButton';
 import Chip from '../Chip';
-import CloseIcon from '../internal/svg-icons/Close';
+import ClearIcon from '../internal/svg-icons/Close';
```

Comment:
```suggestion
import ClearIcon from '../internal/svg-icons/Clear';
```

Reply:
These are aliases, we use Close for the Alert.